contracts/modules/capital/SwapOperator.sol

```diff
@@ -36,7 +40,9 @@ contract SwapOperator is ReentrancyGuard {
   IUniswapV2Router02 constant public router = IUniswapV2Router02(0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D);
   uint constant public MAX_LIQUIDITY_RATIO = 0.015 ether;
   uint constant public MIN_TIME_BETWEEN_SWAPS = 10 minutes;
-
+  address constant public enzymeV4VaultProxyAddress = 0x27F23c710dD3d878FE9393d93465FeD1302f2EbD;
+  IEnzymeV4DepositWrapper enzymeV4DepositWrapper  = IEnzymeV4DepositWrapper(0x4Ffd9cb46F129326efCe0BD30064740Bb79dF6DB);
```

Comment:
How likely is this to change?

Is it better to read it dinamically from another smart contract?

Reply:
It can change. If you have any place in storage with updateable constants it would be better placed there in my opinion